openapi.yaml

```diff
@@ -1660,6 +1660,17 @@ paths:
                   - processes
                   - links
                 properties:
+                  version:
+                    type: string
+                    nullable: true
+                    description: >-
+                      Version number of the openEO process specification this back-end
+                      implements.
```

Comment:
instead of "implements" maybe say "targets" or "uses as reference". 
I wonder if any backend fully implements the full scope of openEO processes. It would be unfortunate if no backend would set this, just because they don't implement everything or have minor deviations in experimental processes.

The main goal of this version identifier, I think, is to communicate what conventions are followed (e.g. v1 raster cubes vs v2 data cubes), or what reference is used to decide about conflicts/differences between versions.

Reply:
```suggestion
                      targets.
```